packages/webapp/next.config.js

```diff
@@ -67,6 +67,10 @@ module.exports = withTM(
         CURRENT_VERSION: version,
       },
       rewrites: () => [
+        {
+          source: '/api/sitemaps/:path*',
+          destination: `${process.env.NEXT_PUBLIC_API_URL}/sitemaps/:path*`,
+        },
```

Comment:
Ah did I break this when removing `/api` rewrite?

Reply:
hehe yes. this was hard to tell, now we have a clear rule

Reply:
:hidethepain: